conf/unbound.conf

```diff
@@ -0,0 +1,68 @@
+server:
+      # the working directory.
+      directory: "/etc/unbound"
+
+      # run as the unbound user
+      username: unbound
+
+      verbosity: 0      # uncomment and increase to get more logging.
+      # logfile: "/var/log/unbound.log" # won't work due to apparmor
+      # use-syslog: no
```

Comment:
Coming back to this PR...

Is unbound logging to somewhere?

Reply:
It's logging to the systemd journal and to the syslog.